examples/rh-sso/rh-sso.conf

```diff
@@ -0,0 +1,20 @@
+server {
+    listen 8180;
+    
+    server_name _;
+
+    set_by_lua $rhsso_endpoint 'return os.getenv("RHSSO_ENDPOINT")';
+
+    location / {
+      content_by_lua_block { require('client_registrations.webhook_handler').call() }
+    }
+
+    # Proxying RH-SSO resources (stylesheets etc.)
+    location ~ ^/auth/resources {
+      proxy_pass $rhsso_endpoint;
```

Comment:
apicast does not have and won't have `resolver` defined, so I'm not sure how this can work